src/client/App.tsx

```diff
@@ -57,6 +58,7 @@ const App: React.FC = () => {
             <Routes>
               <Route path="/" element={<Dashboard />} />
               <Route path="/brokers" element={<Brokers />} />
+              <Route path="/topics" element={<Topics />} />
```

Comment:
We have two routes that are currently the same, please update the path before merge